src/client/containers/OrderConfirmationPage/OrderConfirmationPage.styles.css

```diff
@@ -1,3 +1,57 @@
-div {
-  color: #687808;
+.confirmation-page {
+    align-items: center;
+    margin: 4%;
 }
+
+.confirmation-page .inner-div {
+    background-color: #F6F8EB;
+}
+
+.confirmation-page .wrapper2 {
+    text-align: center;
+    margin-right: 30%;
+}
+
+.confirmation-page .wrapper {
+    width: 90%;
+    text-align: center;
+    padding-top: 8%;
+}
+
+.confirmation-page .wrapper-content {
```

Comment:
This box is harder to name, it seems used in two places, both of which are boxes, `box-content` isn't much better but at least it would line up with the names of the wrapping classes.

Reply:
Updated.. Thank you